SimCalorimetry/EcalSimAlgos/interface/EcalCoder_Ph2.h

```diff
@@ -0,0 +1,83 @@
+#ifndef SimCalorimetry_EcalSimAlgos_EcalCoder_Ph2_h
+#define SimCalorimetry_EcalSimAlgos_EcalCoder_Ph2_h
+
+#include "CalibFormats/CaloObjects/interface/CaloTSamples_Ph2.h"
+#include "CondFormats/EcalObjects/interface/EcalLiteDTUPedestals.h"
+#include "CondFormats/EcalObjects/interface/EcalIntercalibConstantsMC.h"
+#include "CondFormats/EcalObjects/interface/EcalCATIAGainRatios.h"
+#include "SimCalorimetry/EcalSimAlgos/interface/EcalCorrelatedNoiseMatrix_Ph2.h"
+#include "CondFormats/EcalObjects/interface/EcalConstants.h"
+
+template <typename M>
+class CorrelatedNoisifier;
+class EcalLiteDTUSample;
+class EcalDataFrame_Ph2;
+class DetId;
+class EcalLiteDTUPed;
+
+#include <vector>
+
+namespace CLHEP {
+  class HepRandomEngine;
+}
+
+class EcalCoder_Ph2 {
```

Comment:
while this class seems appropriately separate from the phase 0 version (since it uses different records, different calculations, etc.), the "_Ph2" naming convention is somewhat unusual. It might be better to use something more physically meaningful, e.g. derived from any names used in the new electronics.

Reply:
I renamed EcalCoder_Ph2 into EcalLiteDTUCoder